.travis.yml

```diff
@@ -1,17 +1,19 @@
-sudo: false
+sudo: required
```

Comment:
Why do we need `sudo`?

Reply:
Ah, let me try removing that. We needed it for the sandbox stuff.

Reply:
@rwjblue removed.